index.js

```diff
@@ -285,7 +292,7 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 				callback();
 			}.bind(this));
 		}.bind(this));
-		compilation.plugin("additional-assets", function(callback) {
+		compilation.plugin("before-chunk-assets", function() {
```

Comment:
this change is the source of problem I mentioned in https://github.com/webpack-contrib/extract-text-webpack-plugin/pull/457#issuecomment-295194719